api/elastic.py

```diff
@@ -70,6 +72,10 @@ def generateSearchQuery(self, params):
                 searchClauses.append(self.subjectQuery(escapedQuery))
             elif field == 'viaf' or field == 'lcnaf':
                 searchClauses.append(self.authorityQuery(field, escapedQuery))
+            elif field == 'identifier':
+                searchClauses.append(self.identifierQuery(escapedQuery))
+            elif field == 'authority: identifier':
+                searchClauses.append(self.authIdentQuery(authorityList, escapedQuery))
```

Comment:
I think we can/should roll this into one method. The `authority: identifier` will be hard to pass in as part of a parameter string and probably less user friendly. I think it's a relatively simple update that I'll outline below.